src/main/java/org/terasology/namegenerator/creature/CreatureAssetTheme.java

```diff
@@ -16,13 +16,13 @@
 
 package org.terasology.namegenerator.creature;
 
-import java.util.Collections;
-import java.util.List;
-
 import org.terasology.asset.Assets;
 import org.terasology.entitySystem.prefab.Prefab;
 import org.terasology.namegenerator.data.NameGeneratorComponent;
 
+import java.util.Collections;
```

Comment:
This may well be a non-issue. I just pulled the code, attempted a reformat in IntelliJ, and it didn't change this at all. Maybe the old version was out of whack and this simply fixes it?